sdk/identity/identity/src/msal/flows.ts

```diff
@@ -17,6 +17,10 @@ export interface MsalFlowOptions {
   authorityHost?: string;
   authenticationRecord?: AuthenticationRecord;
   disableAutomaticAuthentication?: boolean;
+  knownAuthorities?: Array<string>;
```

Comment:
Once again probably better to have string[]

Reply:
agree